tests/ui/pages/desktop/home.py

```diff
@@ -39,3 +90,25 @@ def users(self):
                 """Number of users that have downloaded the extension"""
                 users_str = self.find_element(*self._users_locator).text
                 return int(users_str.split()[0].replace(',', ''))
+
+    class FeaturedThemes(Region):
+
+        _featured_themes_see_all_link = (
+            By.CSS_SELECTOR, "#featured-themes h2 a")
+        _featured_themes_title_locator = (
+            By.CSS_SELECTOR, "#featured-themes h2")
+        _featured_themes_items_locator = (
+            By.CSS_SELECTOR, "#featured-themes li")
+
+        @property
+        def _wait_for_themes_page(self):
+            self.wait.until(lambda _: 'themes' in self.selenium.current_url)
```

Comment:
This is not used, so let's delete it. What we would want here is a `Themes` page with an appropriate `wait_for_page_to_load`. Then, any interaction that expected this page to load would simply instantiate the object, call this method, and return the new page object. For example:

```python
el.click()
return Themes(...).wait_for_page_to_load()
```